bzlib.rs

```diff
@@ -182,7 +182,7 @@ pub(crate) const BZ_N_OVERSHOOT: usize = (BZ_N_RADIX + BZ_N_QSORT + BZ_N_SHELL +
 pub(crate) const FTAB_LEN: usize = u16::MAX as usize + 2;
 
 pub(crate) struct EState {
-    pub strm: *mut bz_stream,
+    pub strm: usize, // Only for a consistency check
```

Comment:
we could rename this to `strm_addr` then?

Reply:
Sure